src/h5web/toolbar/controls/ScaleSelector/ScaleSelector.tsx

```diff
@@ -1,27 +1,24 @@
 import { ScaleType } from '../../../vis-packs/core/models';
 import Selector from '../Selector/Selector';
-import styles from './ScaleSelector.module.css';
 import ScaleOption from './ScaleOption';
 
 interface Props {
-  value: ScaleType;
   label?: string;
+  value: ScaleType;
   onScaleChange: (scale: ScaleType) => void;
 }
 
 function ScaleSelector(props: Props) {
-  const { value, label, onScaleChange } = props;
+  const { label, value, onScaleChange } = props;
 
   return (
-    <div className={styles.root}>
-      {label && <span className={styles.label}>{label}</span>}
-      <Selector
-        value={value}
-        onChange={onScaleChange}
-        options={Object.values(ScaleType)}
-        optionComponent={ScaleOption}
-      />
-    </div>
+    <Selector
+      label={label}
```

Comment:
Moving the label feature to `Selector`, as it seems to make sense and allows removing `ScaleSelector.module.css`.